services/app-api/utils/validation/schemaMap.test.ts

```diff
@@ -1,6 +1,13 @@
 import { MixedSchema } from "yup/lib/mixed";
 import { AnyObject } from "yup/lib/types";
-import { number, ratio, date, isEndDateAfterStartDate } from "./schemaMap";
+import {
+  number,
+  ratio,
+  date,
+  isEndDateAfterStartDate,
+  nested,
+} from "./schemaMap";
+import {} from "./validation";
```

Comment:
Question: does this do anything? I believe it would run any global code in validation.ts, but I don't think there is any?

Reply:
I removed it! Thanks for catching that 💯